srtp_writer_future.go

```diff
@@ -0,0 +1,95 @@
+// +build !js
+
+package webrtc
+
+import (
+	"io"
+	"sync/atomic"
+
+	"github.com/pion/rtp"
+	"github.com/pion/srtp"
+)
+
+// srtpWriterFuture blocks Read/Write calls until
+// the SRTP Session is available
+type srtpWriterFuture struct {
+	rtpSender      *RTPSender
+	rtcpReadStream atomic.Value // *srtp.ReadStreamSRTCP
+	rtpWriteStream atomic.Value // *srtp.WriteStreamSRTP
+}
+
+func (s *srtpWriterFuture) init() error {
```

Comment:
I just checked the implementations of all methods called from `init()` and it seems that it won't be a big deal if `init` is called multiple times (e.g. by calling both `Read()` and `Write` before the `srtpReady` is closed).

Just wanted to double-check if my understanding is correct?

Reply:
Yea wouldn't be a problem at all!

Reply:
Thanks!